core/lib/spree/core/importer/product.rb

```diff
@@ -7,8 +7,8 @@ class Product
         def initialize(product, product_params, options = {})
           @product = product || Spree::Product.new(product_params)
 
-          @product_attrs = product_params
-          @variants_attrs = options[:variants_attrs] || []
+          @product_attrs = product_params.to_h
+          @variants_attrs = (options[:variants_attrs] || []).map(&:to_h)
```

Comment:
Lower in this file there is

```
{ product: product }.merge(variant_attribute)
```

Where `variant_attribute` is a hash derived from `variants_attrs`. This causes a warning in rails 5 because it is converting an `ActionController::Parameters` to a `Hash` without sanitizing.